src/main/java/com/eviive/personalapi/controller/SkillController.java

```diff
@@ -87,8 +82,16 @@ public ResponseEntity<SkillDTO> save(@RequestBody @Valid final SkillDTO skillDTO
         summary = "Create a skill with an image",
         responses = {
             @ApiResponse(responseCode = "201", description = "Created"),
-            @ApiResponse(responseCode = "400", description = "Bad Request"),
-            @ApiResponse(responseCode = "415", description = "Unsupported Media Type")
+            @ApiResponse(
+                responseCode = "400",
+                description = "Bad Request",
+                content = @Content(schema = @Schema(implementation = ProblemDetail.class))
+            ),
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'annotation array initialization' child has incorrect indentation level 12, expected level should be 16.